### PR TITLE
End of my capstone project one(fastdevz company)

### PR DESCRIPTION
**Pull request by soulemanou-software**
In this PR,

- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @soulemanou-software , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not passing linters, kindly address the linters error you have by checking out this file, https://github.com/soulemanou-software/Capstone--pro--1/blob/desk1/test.html that this the file which is causing you linters errors, actually it's because is an invalid HTML, if you need it in your codes, kindly make sure that it follows HTML5 structure

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,96 +1,130 @@
...
+      <div class="heroCont">
+        <div class="mainHeading">
+          <h2>"Welcome To The World of Infinity Creations"</h2>
+          <h1><br>SOFTWARE IS A GREAT COMBINATION BETWEEN ARTISTRY AND ENGINEERING.<br>2022</h1>
+        </div>
+        <div class="message">
+          <p>
+            Excellent customer service is the number one job in any company! It is the personality of the company and the reason customers come back. Without customers there is no company. (the customer is king at fastdevz)
+          </p>
+        </div>
```
Comment (phelian23) [inline index.html:59]: - [x] [Optional] There is no need wrapping a `<p>` tag in a `<div>` container as the`<p>` tag is also a block-level element.

File: README.md [hunk 2]
```diff
@@ -21,7 +22,8 @@
 ## Live Demo
```
Comment (phelian23) [inline README.md:22]: - [x] Kindly check your deployment link. It seems the link is for an older version of your app.

File: about.html [hunk 3]
```diff
@@ -1,111 +1,100 @@
...
+      <div class="mainMenu">
+        <div class="mainMenuCont">
+          <div class="logo">
+            <a href="index.html"><img src="Resources/images/my-logo.png" alt=""></a>
+          </div>
+          <i id="menuBtn" class="fas fa-bars fa-lg"></i>
+          <nav id="mainNav">
+            <i id="closeBtn" class="fas fa-times fa-lg"></i>
+            <ul>
+              <li><a class="aboutLink" href="about.html">About</a></li>
```
Comment (phelian23) [inline about.html:36]: - [x] It'll be nice if, on the about page, instead of the `about` nav link, you have a `home` nav link.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @soulemanou-software, 

Good job so far!
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!

## Highlight
Good commit message ✔️
No Linter errors ✔️
Follow Github flow ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 3 above)
Comment (soulemanou-software) [reply about.html:36]: done

File: README.md (hunk 2 above)
Comment (soulemanou-software) [reply README.md:22]: hello Mr @phelian23 i have fixed all the required error can i have a reviews again please

File: about.html (hunk 3 above)
Comment (soulemanou-software) [reply about.html:36]: @phelian23 all done here

File: index.html (hunk 1 above)
Comment (soulemanou-software) [reply index.html:59]: @phelian23 is also done here I removed the p tag

**Review by danielufeli — APPROVED** [review]
Hi @soulemanou-software, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**